azext_iot/tests/generators.py

```diff
@@ -68,5 +69,16 @@ def create_c2d_receive_response(minimum=False):
     return baseline
 
 
-def generate_generic_id():
+def generate_generic_id() -> str:
     return str(uuid4()).replace("-", "")
+
+
+def generate_names(prefix: str = "", count: int = 1, max_length: int = 48) -> Union[str, List[str]]:
```

Comment:
We have some other name generation functions also some relying on azure cli testsdk would be nice to align.

Reply:
was going to deal with those when I update/fixturize them